packages/trpc/server/routers/viewer/bookings.tsx

```diff
@@ -268,6 +268,14 @@ export const bookingsRouter = router({
           },
           status: true,
           paid: true,
+          payment: {
```

Comment:
This Query is too big nowadays - needs refactoring (potentially design wise, even) to optimise this query.

Reply:
As discussed, ignore for now.